.github/workflows/webui-periodic.yml.j2

```diff
@@ -56,54 +56,105 @@ jobs:
           repository: rhinstaller/tplib
           path: tplib
 
-      # Use static URL to unpacked image
-      # TODO: unpack the image as the part of the workflow
-      - name: Set the image to be tested
-        id: prepare_repo
-        run:
-          echo "repo_url=$STATIC_REPO_URL" >> $GITHUB_OUTPUT
+      - name: Build anaconda-rpm container (for RPM build)
+        working-directory: ./anaconda
+        run: |
+          # set static tag to avoid complications when looking what tag is used
+          make -f Makefile.am anaconda-rpm-build CI_TAG=$CONTAINER_TAG
+
+      - name: Build Anaconda RPM files
+        working-directory: ./anaconda
+        run: |
+          # output of the build will be stored in ./result/build/01-rpm-build/*.rpm
+          make -f Makefile.am container-rpms-scratch CI_TAG=$CONTAINER_TAG
+          mkdir -p ./anaconda_rpms/
+          cp -av ./result/build/01-rpm-build/*.rpm ./anaconda_rpms/
+
+      - name: Build anaconda-iso-creator container image
+        working-directory: ./anaconda
+        run: |
+          # set static tag to avoid complications when looking what tag is used
+          sudo make -f Makefile.am anaconda-iso-creator-build CI_TAG=$CONTAINER_TAG
+
+      - name: Build the boot.iso
+        working-directory: ./anaconda
+        run: |
+          mkdir -p images
+          # /var/tmp tmpfs speeds up lorax and avoids https://bugzilla.redhat.com/show_bug.cgi?id=1906364
+          sudo podman run -i --rm --privileged \
+            --tmpfs /var/tmp:rw,mode=1777 \
+            -v `pwd`/anaconda_rpms:/anaconda-rpms:ro \
+            -v `pwd`/images:/images:z \
+            --entrypoint /lorax-build-webui \
+            $ISO_BUILD_CONTAINER_NAME:$CONTAINER_TAG
+
+      - name: Clean up after lorax
+        if: always()
+        run: |
+          # remove container images together with the container
+          sudo podman rmi -f $ISO_BUILD_CONTAINER_NAME:$CONTAINER_TAG || true
+          sudo podman rmi -f $RPM_BUILD_CONTAINER_NAME:$CONTAINER_TAG || true
 
       - name: Create Permian settings file
         working-directory: ./permian
         run: |
           cat <<EOF > settings.ini
           [AnacondaWebUI]
+          anaconda_repo=file://${{ github.workspace }}/anaconda
           hypervisor_vm_limit=${{ env.TEST_JOBS }}
           test_timeout=${{ env.LAUNCHER_TIMEOUT_MINUTES }}
           [library]
-          directPath=${{ github.workspace }}/anaconda/ui/webui/test/integration
+          directPath=${{ github.workspace }}/anaconda/ui/webui/test/end2end
           EOF
 
-      - name: Run webui integration tests
+      - name: Run webui end2end tests
         working-directory: ./permian
         run: |
           PYTHONPATH=${PYTHONPATH:-}:${{ github.workspace }}/tplib \
           ./pipeline --debug-log permian.log \
             --settings settings.ini \
-            --override workflows.dry_run=False \
+            -o github.pull-request=${{ github.event.issue.number }} \
```

Comment:
github-pr report sender is not used (there is no pull request) so these options are not doing anything

Reply:
Updated